app/models/concerns/waste_carriers_engine/can_have_registration_attributes.rb

```diff
@@ -86,12 +86,27 @@ module CanHaveRegistrationAttributes
       field :total_fee,                                        type: String
 
       scope :search_term, lambda { |term|
-        escaped_term = Regexp.escape(term) if term.present?
+        if term.present?
```

Comment:
Swapping this for `return unless term.present?` could reduce the indentation here